### PR TITLE
Update README, in preparation for GA release

### PR DESCRIPTION
**Pull request by jasonblais**
cc: @cpoile @levb @cpanato

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+2. Create an AWS SNS Topic with an HTTPS subscription to https://your-mattermost-url/plugins/com.mattermost.aws-sns?token=your-mattermost-token`, where `your-mattermost-url` refers to your Mattermost URL, and `your-mattermost-token` was generated on a previous step. [Follow this documentation](https://docs.safe.com/fme/html/FME_Server_Documentation/ReferenceManual/Amazon_SNS_Publisher_Configure_AWS_Subscription.html) for additional configuration options.
 
-  - Install the plugin in your Mattermost instance
-  - Configure the plugin
-    - Set the `Team` and `Channel` that you want to receive the messages. If the Channel does not exist the plugin will create the channel for you.
-    Set the `Team` and `Channel` with comma, ie. `teamb,channelx`.
-  - Set the `AllowedUserIds` with the users ids, this will allow specific users to approve the subscription from AWS SNS.
-  - Set the `Username`, this will be user that will post the messages.
-  - Set the `Token`, this will be use when creating the SNS subscription.
+3. Switch to the Mattermost channel you configured to receive notifications. Accept the subscription posted to the channel.
```
Comment (jasonblais) [inline README.md]: I'm not sure if this step is accurately described. I didn't (yet at least) have time to set up a CloudWatch account and test this user experience.

Comment (cpoile) [reply README.md]: Me neither, sorry I can't be of more help

Comment (cpanato) [reply README.md]: When you configure the AWS SNS to send the messages to MM, AWS SNS will send a message to confirm the subscription, this message will arrive in the channel you selected to receive the messages and then you need to confirm the subscription clicking in the action button

Comment (jasonblais) [reply README.md]: Thank you! Updated the note accordingly

**Pull request merged**